entities/subscribersevent.go

```diff
@@ -0,0 +1,23 @@
+package entities
+
+import (
+	"time"
+
+	"github.com/segmentio/ksuid"
+)
+
+const (
+	SubscriberEventTypeCreated      EventType = "created"
+	SubscriberEventTypeDeleted      EventType = "deleted"
+	SubscriberEventTypeUnsubscribed EventType = "unsubscribed"
+)
+
+// SubscribersEvent represents an event saved on subscriber's change
+type SubscribersEvent struct {
+	ID              ksuid.KSUID `json:"id" gorm:"column:id; primary_key:yes"`
+	UserID          int64       `json:"user_id"`
+	SubscriberID    int64       `json:"subscriber_id"`
```

Comment:
@gudgl @FilipNikolovski I think we don't need sub id since we are going to store removed subs too.

Reply:
Yes if it is deleted than we dont have the id anymore